packages/mask/src/plugins/Wallet/services/legacyWallet.ts

```diff
@@ -84,6 +84,31 @@ export async function getLegacyWallets(provider?: ProviderType) {
     }
 }
 
+export async function getLegacyWalletRecords() {
```

Comment:
Why not use `getLegacyWallets`? You repeated that function

Reply:
`getLegacyWallets` is reserved for JS to fetch connected wallets information from various providers.
The newly added `getLegacyWalletRecords` only returns saved wallets information from indexDB, currently it's used for mobile apps to migrate legacy wallets to native wallets.

Reply:
You can add a new parameter to opt-in to always get data from indexedDB instead of copy-paste the function body